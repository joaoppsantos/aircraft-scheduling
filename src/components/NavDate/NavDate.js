const NavDate = () => {
    return (
        <div>
            Nav Date
        </div>
    );
}

export default NavDate;
