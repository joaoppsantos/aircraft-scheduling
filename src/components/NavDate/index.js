export { default as NavDate } from './NavDate';
