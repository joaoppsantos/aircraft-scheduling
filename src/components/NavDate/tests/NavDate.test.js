import * as React from 'react';
import { render } from '@testing-library/react';
import NavDate from '../NavDate';

beforeAll(() => {
    jest.useFakeTimers('modern');
    jest.setSystemTime(new Date(2022, 1, 1));
});

afterAll(() => {
    jest.useRealTimers();
});

describe('<NavDate />', () => {
    it('should render correctly', () => {
        
        const { container } = render(
            <NavDate />
        );

        expect(container).toMatchSnapshot();
    });
});
