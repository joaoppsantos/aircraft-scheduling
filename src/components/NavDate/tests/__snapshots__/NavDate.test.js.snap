// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavDate /> should render correctly 1`] = `
<div>
  <div>
    Tue, February 1, 2022
  </div>
</div>
`;
